### PR TITLE
chore: update README with updated links und up to date documentation

### PR DESCRIPTION
**Pull request by sjschlapbach**

### CODE REVIEW

**Comment by sjschlapbach** [comment]
TODO: probably we should also replace the title image to an updated nice image from KlickerUZH v3.0 for consistency

File: README.md [hunk 1]
```diff
@@ -1,54 +1,61 @@
...
 
-> Please note that this is the repository branch for the upcoming KlickerUZH v3.0 with a beta launch scheduled before the spring semester 2023 and stable release for the autumn semester 2023. If you would like to have more information on the new concept of v3.0, have a look at our [public discussion](https://community.klicker.uzh.ch/t/klickeruzh-v3-0-concept-and-request-for-feedback/79).
+> Please note that this is the repository branch for KlickerUZH v3.0 released in August 2023. If you would like to have more information on the new concept of v3.0, have a look at our [public discussion](https://community.klicker.uzh.ch/t/klickeruzh-v3-0-concept-and-request-for-feedback/79).
 >
-> Please navigate to the [dev branch](https://github.com/uzh-bf/klicker-uzh/tree/dev) for the currently stable production code.
+> Please navigate to the [dev branch](https://github.com/uzh-bf/klicker-uzh/tree/dev) for the older [KlickerUZH v2.0](https://app.klicker.uzh.ch/), which will be available until the end of 2023. To migrate to KlickerUZH v3.0, first [log in](https://auth.klicker.uzh.ch/) and start the [migration process](https://manage.klicker.uzh.ch/migration).
```
Comment (rschlaefli) [inline README.md]:
```suggestion
> Please navigate to the [dev branch](https://github.com/uzh-bf/klicker-uzh/tree/dev) for the older [KlickerUZH v2.0](https://app.klicker.uzh.ch/), which will be available until the end of 2023. To migrate to KlickerUZH v3.0, start the [migration process](https://manage.klicker.uzh.ch/migration).
```

File: README.md [hunk 2]
```diff
@@ -1,54 +1,61 @@
...
+- [Fontend Authentication](https://github.com/uzh-bf/klicker-uzh/tree/v3/apps/auth) is the authentication frontend of KlickerUZH, providing login functionalities through Edu-ID accounts and delegated logins to the manage frontend.
+- [Backend Docker](https://github.com/uzh-bf/klicker-uzh/tree/v3/apps/backend-docker) is the main backend service of KlickerUZH.
+- [Backend Responses](https://github.com/uzh-bf/klicker-uzh/tree/v3/apps/func-incoming-responses) is a service that handles incoming student responses during a live session and puts them into an Azure queue for improved load handling.
+- [Backend Response Processor](https://github.com/uzh-bf/klicker-uzh/tree/v3/apps/func-response-processor) accesses queued elements from the aforementioned service and processes them by computing scores and experience points, updating the cache, etc.
+- The [v2.0 Export](https://github.com/uzh-bf/klicker-uzh/tree/v3/apps/func-migration-v2-export) and [v3.0 Import](https://github.com/uzh-bf/klicker-uzh/tree/v3/apps/func-migration-v3-import) folders contain code related to the migration from KlickerUZH v2.0 to v3.0.
```
Comment (rschlaefli) [inline README.md]:
```suggestion
- The [v2.0 Export](https://github.com/uzh-bf/klicker-uzh/tree/v3/apps/func-migration-v2-export) and [v3.0 Import](https://github.com/uzh-bf/klicker-uzh/tree/v3/apps/func-migration-v3-import) folders contain serverless Azure Functions related to the migration from KlickerUZH v2.0 to v3.0.
```

File: README.md [hunk 3]
```diff
@@ -1,54 +1,61 @@
...
 
-In addition to key application components, this repository also includes the codebases for our landing page and documentation (www.klicker.uzh.ch), as well as deployment scripts and examples:
+In addition to the key application components, this repository also includes the codebases for our landing page and documentation at [www.klicker.uzh.ch](https://www.klicker.uzh.ch/), as well as deployment scripts. An updated deployment documentation for self-hosting KlickerUZH v3.0 will be added later:
```
Comment (rschlaefli) [inline README.md]:
```suggestion
In addition to the key application components, this repository also includes the codebases for our landing page and documentation at [www.klicker.uzh.ch](https://www.klicker.uzh.ch/), as well as deployment scripts for Helm/Kubernetes. An updated deployment documentation for self-hosting KlickerUZH v3.0 will be added until the end of the year.
```

File: README.md [hunk 4]
```diff
@@ -1,54 +1,61 @@
...
 
-- [Grading](https://github.com/uzh-bf/klicker-uzh/tree/v2/packages/grading): The grading package provides the grading logic that is used to assign scores to participants and groups in live sessions, learning elements, micro-sessions and other KlickerUZH elements.
-- [Markdown](https://github.com/uzh-bf/klicker-uzh/tree/v2/packages/markdown): The markdown package exports a React component to render markdown strings into formatted elements.
-- [Shared Components](https://github.com/uzh-bf/klicker-uzh/tree/v2/packages/shared-components): The `shared-components` package is configured as an internal turborepo package, mainly providing the possibility to share components between the two frontends and reduce code duplication.
+- [Prisma](https://github.com/uzh-bf/klicker-uzh/tree/v3/packages/prisma): The prisma package contains the SQL database schema as well as migration scripts between different minor versions of KlickerUZH.
+- [GraphQL](https://github.com/uzh-bf/klicker-uzh/tree/v3/packages/graphql): The GraphQL package contains the complete GraphQL schema and all resolvers for the KlickerUZH backend. For consistent typing, Pothos GraphQL and an auto-generated schema are used.
```
Comment (rschlaefli) [inline README.md]:
```suggestion
- [GraphQL](https://github.com/uzh-bf/klicker-uzh/tree/v3/packages/graphql): The GraphQL package contains the complete GraphQL schema and all resolvers for the KlickerUZH backend, as well as all business logic for provided services. For consistent typing, Pothos GraphQL and an auto-generated schema are used.
```

File: README.md [hunk 5]
```diff
@@ -1,54 +1,61 @@
...
+- [Grading](https://github.com/uzh-bf/klicker-uzh/tree/v3/packages/grading): The grading package provides the grading logic that is used to assign scores and experience points to participants and groups in gamified live sessions, learning elements, micro-sessions and other KlickerUZH elements.
+- [LTI](https://github.com/uzh-bf/klicker-uzh/tree/v3/packages/lti): The LTI package contains logic to offer automatic sign ins for students through LTI (OLAT and Moodle integration).
```
Comment (rschlaefli) [inline README.md]:
```suggestion
- [LTI](https://github.com/uzh-bf/klicker-uzh/tree/v3/packages/lti): The LTI package contains logic to offer automatic sign ins for students through the Learning Tools Interoperability "LTI" standard (for OpenOLAT and Moodle integration).
```

File: README.md [hunk 6]
```diff
@@ -1,54 +1,61 @@
...
+- [Internationalization](https://github.com/uzh-bf/klicker-uzh/tree/v3/packages/i18n): The i18n package provides internationalization support across all KlickerUZH frontends.
```
Comment (rschlaefli) [inline README.md]:
```suggestion
- [Internationalization](https://github.com/uzh-bf/klicker-uzh/tree/v3/packages/i18n): The i18n package provides internationalized messages for all KlickerUZH services.
```

**Review by rschlaefli — APPROVED** [review]

**Pull request merged**